### PR TITLE
update pom to the 2019-09-26 version of financials and get code to co…

### PR DESCRIPTION
**Pull request by db483**
…mpile

### CODE REVIEW

File: src/main/java/edu/cornell/kfs/coa/businessobject/lookup/CuOrganizationRoutingModelNameLookupableHelperServiceImpl.java [hunk 1]
```diff
@@ -27,6 +27,7 @@
 import org.kuali.kfs.kns.lookup.HtmlData.AnchorHtmlData;
 import org.kuali.kfs.kns.web.struts.form.LookupForm;
 import org.kuali.rice.krad.bo.BusinessObject;
+import org.kuali.kfs.krad.util.KRADConstants;
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/coa/businessobject/lookup/CuOrganizationRoutingModelNameLookupableHelperServiceImpl.java]: Is this import needed?

File: src/main/java/edu/cornell/kfs/coa/service/impl/CuAccountServiceImpl.java [hunk 2]
```diff
@@ -14,7 +15,6 @@
 
 public class CuAccountServiceImpl extends AccountServiceImpl implements CuAccountService{
     
-    private static final String DEFAULT_BENEFIT_RATE_CATEGORY_CODE_BY_ACCOUNT_TYPE = "DEFAULT_BENEFIT_RATE_CATEGORY_CODE_BY_ACCOUNT_TYPE";
     private static final String DEFAULT_BENEFIT_RATE_CATEGORY_CODE = "DEFAULT_BENEFIT_RATE_CATEGORY_CODE";
```
Comment (cah292) [inline src/main/java/edu/cornell/kfs/coa/service/impl/CuAccountServiceImpl.java]: Can this constant be removed as well?

File: src/main/java/edu/cornell/kfs/coa/businessobject/lookup/CuOrganizationRoutingModelNameLookupableHelperServiceImpl.java (hunk 1 above)
Comment (db483) [reply src/main/java/edu/cornell/kfs/coa/businessobject/lookup/CuOrganizationRoutingModelNameLookupableHelperServiceImpl.java]: removed

File: src/main/java/edu/cornell/kfs/coa/service/impl/CuAccountServiceImpl.java (hunk 2 above)
Comment (db483) [reply src/main/java/edu/cornell/kfs/coa/service/impl/CuAccountServiceImpl.java]: removed

**Comment by db483** [comment]
[1] new system property added Dlog4j.configurationFile="/Users/danielabalmus/Documents/kfs-config/log4j2.json"; log4j.settings.file to be removed from config file

[2] added

[3] updated

[4] updated

**Comment by db483** [comment]
I have updated the institutional-config.properties file

**Pull request merged**